MaxPowerLevel/Views/Shared/RecommendationsPartial.cshtml

```diff
@@ -2,19 +2,33 @@
 
 @if(Model.Any())
 {
-  <ol>
-    @foreach(var recommendation in Model)
-    {
-      <li>@recommendation</li>
-      @if(recommendation.Activities.Any())
-      {
-        <ol>
-          @foreach(var activity in recommendation.Activities)
-          {
-            <li>@activity</li>
+  <div class="panel recommendations-panel">
+    <p class="panel-heading">
+      Recommendations
+    </p>
+      <div class="panel-block">
+        <table class="table">
+          <tbody>
+          @foreach(var recommendation in Model){
+            <tr>
+              <td>
+                @recommendation
+                @if(recommendation.Activities.Any() && ViewData.Eval("wide") != null) {
```

Comment:
If I'm understanding this correctly, this means that subactivites will only be shown in the character page, and not the All Characters page, right? It might be good to be more granular about this. I think it's fine if some sub activities are hidden (like the vendor recommendations), but others like Pinnacles really should be shown there. Otherwise if all your characters are above the powerful cap all you can see on the All Characters page is "Pinnacle Engrams".

Reply:
I had this for space concerns earlier. I just unhid it and... it looks fine now with some other layout tweaks. Closing this and giving you a new one.